src/paperless/settings.py

```diff
@@ -933,6 +935,19 @@ def default_threads_per_worker(task_workers) -> int:
 if TIKA_ENABLED:
     INSTALLED_APPS.append("paperless_tika.apps.PaperlessTikaConfig")
 
+AUDIT_ENABLED = __get_boolean("PAPERLESS_AUDIT_ENABLED", "NO")
+if AUDIT_ENABLED:
+    INSTALLED_APPS.append("auditlog")
+    MIDDLEWARE.append("auditlog.middleware.AuditlogMiddleware")
+db_conn = connections["default"]
+
+all_tables = db_conn.introspection.table_names()
+
+if ("auditlog_logentry" in all_tables) and not (AUDIT_ENABLED):
+    raise ImproperlyConfigured(
+        "auditlog table was found but PAPERLESS_AUDIT_ENABLED is not active.",
+    )
+
```

Comment:
Why the warning in this case?

Reply:
My thought was if you want/need an audit log there is no point in being able to simply turn it off and on at will. This is a just a very simple way to check that it was ever turned on. Of course there are ways around this that anyone here could do in two seconds but thats not really the point. It's about showing that there is a system in place that means the audit log can be trusted (well somewhat....). I'm still not sure how with a completely self hosted system I could ever prove it has never been changed as any log/encryption/has/etc/etc/etc could be changed by me.

Reply:
TBH, if we go ahead with this feature, I think it should be enabled by default (perhaps disable-able, but even that not sure why). Unless there are downsides to doing so

Reply:
Nothing against the idea (or doing some more work to look less like an idiot as I feel I look right now :sweat_smile:)
My thought was pretty much...if it wasn't a feature already then problably most people don't need it so why add it as default.

Reply:
This isn't something I want to see enabled by default.  It seems like a niche use case and enabled by default is just bloating the database for those who don't have any use for it.

Reply:
ok, lets leave for now, call this mvp

Reply:
Yup it is very minimal but that was the whole idea....it should exist and nothing more